Package.resolved

```diff
@@ -23,17 +23,17 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/groue/GRDB.swift.git",
       "state" : {
-        "revision" : "3df9e8e5c10bd152f02d9aa349a21a0c527e6bda",
-        "version" : "6.2.0"
+        "revision" : "8330469ac3cbbf0ee52b7e8a4e2b1f4c44bb13ea",
+        "version" : "6.4.0"
```

Comment:
Die Version stimmt nicht mit der Information die in der Changelog und im Acknowledgment steht nicht über ein.

Reply:
Verstehe nicht warum das bei mir nicht auf 6.5.0 aktualisiert

Reply:
9271b27fb07c0f015e7aea5e9a82a9d2d2c6dfa6

Ich habe das SDK als Package geöffnet und damit war es sofort erledigt.